main/modes/utilities/gamepad/gamepad.c

```diff
@@ -665,6 +669,226 @@ void gamepadMainLoop(int64_t elapsedUs __attribute__((unused)))
             // barY += (ACCEL_BAR_HEIGHT + ACCEL_BAR_SEP);
         }
 
+        // Display controller state on Eye LED arrays
+        uint8_t bitmap[EYE_LED_H][EYE_LED_W] = {0};
+        uint8_t btnBrightness = 0;
+
+        switch (gamepad->gamepadType)
+        {
+            case GAMEPAD_NS:
+                if ((gamepad->gpNsState.hat == gamepad->previousHat)
+                    && (gamepad->gpNsState.buttons == gamepad->previousButtons))
+                {
+                    // Skip eye update if no change
+                    break;
+                }
+
+                
+                bitmap[5][0]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_TL) ? EYE_LED_BRIGHT : 0;
+                bitmap[5][5]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_TL2) ? EYE_LED_BRIGHT : 0;
+                bitmap[5][6]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_TR2) ? EYE_LED_BRIGHT : 0;
+                bitmap[5][11] = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_TR) ? EYE_LED_BRIGHT : 0;
+
+                bitmap[0][0]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_CAPTURE) ? EYE_LED_BRIGHT : 0;
+                bitmap[0][5]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_MINUS) ? EYE_LED_BRIGHT : 0;
+                bitmap[0][6]  = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_PLUS) ? EYE_LED_BRIGHT : 0;
+                bitmap[0][11] = (gamepad->gpNsState.buttons & GAMEPAD_NS_BUTTON_HOME) ? EYE_LED_BRIGHT : 0;
```

Comment:
> The Switch buttons all worked, but pause and menu didn't light up in PC mode. Are they supposed to?

This section is unique to NS, so it looks like pause and menu aren't captured for PC Controller